app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -407,7 +407,7 @@ public void onNotFound(final String key) {
                             @Override
                             public void success(Person person) {
                                 if (person != null) {
-                                    App.getInstance().getModelCache().put(key, person);
```

Comment:
:+1: We should look into the usages of `put` and make all of them async. The ModelCache causes a lot of issues in strict mode.